File: examples-sdk/swift/022-basic-sidecar-python/main.swift
// 022-basic-sidecar-python — Swift SDK equivalent
// Compile: vil compile --from swift --input 022-basic-sidecar-python/main.swift --release

let server = VilServer(name: "sidecar-python-example", port: 8080)
let fraud = ServiceProcess(name: "fraud")
fraud.endpoint(method: "GET", path: "/status", handler: "fraud_status")
fraud.endpoint(method: "POST", path: "/check", handler: "fraud_check")
server.service(fraud)
let root = ServiceProcess(name: "root")
root.endpoint(method: "GET", path: "/", handler: "index")
server.service(root)
server.compile()

File: examples-sdk/kotlin/022-basic-sidecar-python/main.kt
// 022-basic-sidecar-python — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 022-basic-sidecar-python/main.kt --release

fun main() {
    val server = VilServer("sidecar-python-example", 8080)
    val fraud = ServiceProcess("fraud")
    fraud.endpoint("GET", "/status", "fraud_status")
    fraud.endpoint("POST", "/check", "fraud_check")
    server.service(fraud)
    val root = ServiceProcess("root")
    root.endpoint("GET", "/", "index")
    server.service(root)
    server.compile()
}
